toolkit/tests/api/test_permissions.py

```diff
@@ -0,0 +1,88 @@
+from unittest.mock import MagicMock
+
+from pythonit_toolkit.api.permissions import IsService
+from pythonit_toolkit.headers import SERVICE_JWT_HEADER
+from pythonit_toolkit.pastaporto.tokens import generate_service_to_service_token
+from ward import raises, test
+
+
+@test("Allowed callers pass the permission")
+async def _():
+    test_token = generate_service_to_service_token("test", "gateway", "users-backend")
+    PermissionClass = IsService(["gateway"], "test", "users-backend")
+    headers = {SERVICE_JWT_HEADER: test_token}
+    mock_info = MagicMock()
+    mock_info.context.request.headers.get = headers.get
+    assert PermissionClass().has_permission(None, mock_info) is True
+
+
+@test("Not allowed allowed callers fail the permission")
```

Comment:
Is worth adding a test with a schema Query with this permission?
So an "integration" test to see how this permission can be used in schema...

Reply:
Uhm, I like the idea but we don't have any test like this in the toolkit so I am not sure how hard it would be to setup 🤔